acceptance-tests/domain_fronting_test.go

```diff
@@ -201,6 +186,9 @@ var _ = Describe("Domain fronting", func() {
 			expect200(nonMTLSClient.Do(req))
 			expect421(mtlsClient.Do(req))
 
+			By("Sending a request to HAProxy with a mismatched SNI and Host header it returns a 400")
+			expect400BadRequestNoHostHeader(haproxyInfo.PublicIP, tlsConfig)
```

Comment:
Same here. We only send a request with missing host header.

Reply:
fixed